.gitignore

```diff
@@ -1,15 +1,8 @@
-pkg
-doc/*
-doc/**/*
-*swp
+doc/**
 spec/fixtures/cassettes
 spec/fixtures/credentials.yml
-coverage
-.yardoc
-example.rb
-bin/*.yml
-.sass-cache
-config/mongod
+coverage/**
+.yardoc/**
 Gemfile.lock
-vendor/bundle
-.bundle
+vendor/bundle/**
+
```

Comment:
if you have these files you should remove them in `.gitignore_global`, not in every project